src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14180,6 +14356,41 @@ Label900: ;
 		return StringOut;
 	}
 
+	bool 
+	isNumber(std::string s)
+	{
+		bool is = true;
+		try {
+			std::stof(s);
+		} catch ( std::exception & e ) {
```

Comment:
If trailing whitespace is a concern here is some updated (untested) code to deal with it.
```cpp
bool
isNumber(std::string const & s) {
  char* p;
  strtod( s.c_str(), &p, 10 );
  for ( ; isspace( *p ); ++p ); // handle trailing whitespace
  return *p == 0;
}
```